tools/runners/common_runner.py

```diff
@@ -49,6 +60,69 @@ def get_acounts_with_token(accounts: List[ExportedAccount], token_name: str) ->
     return accounts_with_token
 
 
+def sync_account_nonce(exported_account: ExportedAccount) -> ExportedAccount:
+    """Sync account nonce"""
+    network_providers = NetworkProviders(API, PROXY)
+    account = Account(exported_account.address, config.DEFAULT_OWNER)
+    account.address = WrapperAddress.from_bech32(exported_account.address)
+    account.sync_nonce(network_providers.proxy)
+    exported_account.nonce = account.nonce
+    return exported_account
+
+
+def get_default_signature() -> str:
+    """Get default signature"""
+
+    network_providers = NetworkProviders(API, PROXY)
+    chain_id = network_providers.proxy.get_network_config().chain_id
+    tx_config = DefaultTransactionBuildersConfiguration(chain_id=chain_id)
+    funding_account = Account(address=None, pem_file=config.DEFAULT_OWNER)
+    transaction = EGLDTransferBuilder(
+        config=tx_config,
+        sender=funding_account.address,
+        receiver=funding_account.address,
+        payment=TokenPayment.egld_from_amount(0.001),
+        nonce=funding_account.nonce,
+    ).build()
+    return funding_account.sign_transaction(transaction)
+
+
+def fund_accounts(accounts: List[ExportedAccount]) -> None:
```

Comment:
we should probably name this as `fund_sf_accounts` since it's very specific to shadowfork only

Reply:
done